### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by prasadt1**
Updated the ReadMe.md to add the Overall process for  - ADR Submission / Review and Acceptance into Technical Specification

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,10 +16,42 @@ For this, we use an ADR-inspired process.
 
 To make a meaningful contribution:
 
-1. Create an ADR proposal in the [decisions/log](decisions/log/) with status `In Progress`
-    1. If possible, already create the changes to the technical specification text as well (see above)
-2. Create a Pull Request with the ADR proposal and optionally the specification changes
-3. Once there is consensus on the ADR proposal, the ADR proposal is merged into the `main` branch
+## ADR Submission, Review, and Acceptance Process
+Architecture decisions are submitted via creating a ``Markdown``
```
Comment (bhadley) [inline README.md:20]: Would advise to be more specific than "Architecture decisions", as there is currently confusion in the community regarding what should be raised as an ADR and what shouldn't be. Would advise to specify as "Architecture decisions and changes to the technical specification".

File: README.md [hunk 2]
```diff
@@ -16,10 +16,42 @@ For this, we use an ADR-inspired process.
...
+
+- ADRs are numbered sequentially and monotonically. Numbers are not reused.
+- Format of the ADR: Please see an [existing ADR](https://github.com/wbcsd/data-exchange-protocol/blob/adr-0021-auth-conformance-issues/decisions/log/0015-pagination.md) for the format
```
Comment (bhadley) [inline README.md:23]: Is there a reason why we don't share a "template" for the ADR? Many times in the past few months I have been asked for the ADR template - folks like to copy and paste it, and then fill it out. Currently it is here but think this should be in github and we would eliminate it in .docx https://wbcsd.sharepoint.com/:w:/s/ClimateEnergy/Efozt_RoKPFFsAvQ97B-gKIB1XeC-er3jxuFxTxiJd9-ZA?e=wH9vq2 Would be good to save them time, and also standardized the format of the ADR. Would also recommend including an example in addition to the template.

File: README.md [hunk 3]
```diff
@@ -16,10 +16,42 @@ For this, we use an ADR-inspired process.
...
+
+### Submitting an ADR
+
+- Fork and Clone the repo to your local
+- Create a new branch (name as you prefer but recommend to use your GitHub handle + "-" + adr name e.g. <githubhandle>-adr-0016)
+- Create a Markdown file for the ADR under the [decisions/log](decisions/log/) folder in your local branch (based on the proposed format)
+    -  If possible, submit respective changes (based on your ADR proposal) already to the technical specification text as well (see above) 
+- Commit and push the changes to github repo and raise a PR to merge the forked repo into WBCSD `base repo` (for more information please refer [this link](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)). Tag reviewers as necessary or communicate via email to the concerned audience to get the ADR Review process started
```
Comment (bhadley) [inline README.md:31]: Do we have a process for who should be tagged as reviewers? Not sure how someone raising an ADR would know who to tag as a reviewer.

File: README.md [hunk 4]
```diff
@@ -16,10 +16,42 @@ For this, we use an ADR-inspired process.
...
+  
+- Reviewers will review and provide their feedback as depicted above
+- Incorporate the comments as applicable or reply via inline comments as necessary
+- Once there is consensus on the ADR proposal, update the Status of the ADR to “Accepted". PACT Team will then merge the ADR proposal into the `main` branch
+
+## Propagating Accepted ADR changes to Technical Specification
+- Content from Accepted ADRs will be incorporated in the applicable Technical Specification as below -
+    - If the PR for the ADR **did not already include** proposed changes to the Technical Spectification ([see above](https://github.com/wbcsd/data-exchange-protocol/edit/main/README.md#on-providing-feedback-and-participating-in-the-development-of-the-technical-specifications:~:text=If%20possible%2C%20submit%20respective%20changes%20(based%20on%20your%20ADR%20proposal)%20already%20to%20the%20technical%20specification%20text%20as%20well%20(see%20above))) and was merged into `main` branch -  then a new PR needs to be submitted with the respective changes. Otherwise if the PR was not merged into the `main` branch, please Committ the respective changes in the same PR. <br/><br/>
```
Comment (bhadley) [inline README.md:50]: Typo in sentence, "Committ" should be spelled "Commit"

File: README.md [hunk 5]
```diff
@@ -16,10 +16,42 @@ For this, we use an ADR-inspired process.
 
 To make a meaningful contribution:
 
-1. Create an ADR proposal in the [decisions/log](decisions/log/) with status `In Progress`
-    1. If possible, already create the changes to the technical specification text as well (see above)
-2. Create a Pull Request with the ADR proposal and optionally the specification changes
-3. Once there is consensus on the ADR proposal, the ADR proposal is merged into the `main` branch
+## ADR Submission, Review, and Acceptance Process
```
Comment (bhadley) [inline README.md:19]: Thanks Prasad for updating this process, the updated version is much clearer. I shared a few comments. Also, after we discuss the backlog refinement and feedback process, this ADR process can also be updated again as it should become clearer how ADR feedback is raised vs. all other feedback (i.e. via Notion). Let's discuss!

File: README.md (hunk 2 above)
Comment (raimundo-henriques) [reply README.md:23]: I added a (minimal) template as part of this PR: [decisions/template/adr-template.md](https://github.com/wbcsd/data-exchange-protocol/pull/17/files#diff-ef2d552d6c024a6a955e8e851bcb8a09cf4d3f9bcc2fa93582e66ca86d9fd5c3). 
It is not as comprehensive as the .docx document, however.

**Review by raimundo-henriques — APPROVED** [review]

**Review by trivets-rhodium — APPROVED** [review]

**Review by trivets-rhodium — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (raimundo-henriques) [reply README.md:31]: Jus a comment about this: I approved this PR with the `trivets-rhodium` GitHub account (which is just an account I use for quick GH tests) to make sure that any GH user can review PR's in this repo even without being tagged as a reviewer. 
While tagging is nevertheless important, I guess there's also the possibility of asking stakeholders to review a certain PR even without formally tagging them on GH.

File: README.md (hunk 5 above)
Comment (raimundo-henriques) [reply README.md:19]: Thank you for your comments and suggestions, Beth! I sync with @prasadt1 on this and made the relevant changes. Happy to help in any further changes and/or improvements!

Note: the review by `trivets-rhodium` is mine and (as explained in another comment) was just a quick test.

Comment (bhadley) [reply README.md:19]: Nice! Thanks @raimundo-henriques!

**Review by bhadley — APPROVED** [review]
Looks good to me - and we can evolve over time!

**Pull request merged**